### ISSUE TITLE
Speed improvements

### ISSUE THREAD

**Issue by J05HI** [jcgruenhage/PlayMusicExporter#51]
### Some stuff could be improved:

- [ ] Currently each cover and music track is loaded separately with a superuser command which is really slow. 
We should copy all covers and encrypted mp3s to a normal accessible location (if they aren't already) 
  - [ ] Cover loading
  - [ ] Music loading

**Comment by jcgruenhage** [comment]
> We should copy all covers and encrypted mp3s

Wow, NO. When would you do that? During app startup? That would take forever, ignoring the fact, that this will use a shitload of additional space. *Maybe covers*, no way we'll copy mp3s though.

**Comment by J05HI** [comment]
@jcgruenhage 

No, sorry for the bad description of the issue. I won't copy the mp3s. Only the covers (this takes a part of a second) and doesn't take that much space. 

We only need to copy them if they aren't on the SD card, if they are already (in "SDCARD/Android/data...") we can access them with a normal file access. 

Loading the covers on scroll will nearly instantly then.

**Comment by jcgruenhage** [comment]
That seems more reasonable, thanks for the better description. Should we maybe have thumbnails for most covers in memory already? Not large ones, just 128² or something like that.

**Comment by J05HI** [comment]
@jcgruenhage 

Maby if it doesn't take too much memory.
